### PR TITLE
Update debugging-main-process and Add planned-breaking-changes

### PR DESCRIPTION
**Pull request by tinydew4**

### CODE REVIEW

**Comment by zeke** [comment]
Since the new `planned-breaking-changes.md` file appears to be an unchanged/untranslated copy of the English version, maybe we should just link to the English version?

**Comment by tinydew4** [comment]
Hey @zeke thanks for let me know.. but could you tell me which part or line is unchanged/untranslated? I need more specific info.

File: docs-translations/ko-KR/tutorial/planned-breaking-changes.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
+# API 변경 계획
+
+다음은 Electron 2.0 에서 제거될 API 목록입니다.
+
+정해진 적용 계획은 없지만 적어도 90일 전에 중단 경고를 할 것 입니다.
```
Comment (zeke) [inline docs-translations/ko-KR/tutorial/planned-breaking-changes.md]: Maybe just keep this top part ☝️ in Korean, then remove everything after it, replacing it with a link to the English list of deprecations:

```
[breaking changes](docs/tutorial/breaking-changes.md]
```

**Comment by zeke** [comment]
Looks good to me!

**Comment by tinydew4** [comment]
Thanks @zeke .

**Comment by zcbenz** [comment]
:+1:

**Pull request merged**